external/netfx/netfx.depproj

```diff
@@ -25,5 +25,11 @@
     <FileToExclude Include="System.EnterpriseServices.Thunk" />
     <FileToExclude Include="System.EnterpriseServices.Wrapper" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)'=='uap' or '$(TargetGroup)'=='uapaot'">
```

Comment:
No you shouldn't need them in this depproj any more we need to keep the set of things resolve across the different configurations consistent otherwise BuildAll will keep thrashing this directory.